### PR TITLE
BCM2708: DT: change 'axi' nodename to 'soc'

### PR DESCRIPTION
**Pull request by notro**
Change DT node named 'axi' to 'soc' so it matches ARCH_BCM2835.
The VC4 bootloader fills in certain properties in the 'axi' subtree,
but since this is part of an upstreaming effort, the name is changed.

Signed-off-by: Noralf Tronnes notro@tronnes.org

### CODE REVIEW

**Comment by popcornmix** [comment]
Should firmware replace:

```
/axi/vc_mem/reg
/axi/usb/hub/ethernet/mac-address
/axi/dma/broadcom,channels
/axi/sdhci/clock-frequency
/axi/uart0/clock-frequency
```

with /soc/ ?

Do any machine/driver files need updating?

**Comment by notro** [comment]
The clocks are in the clocks subtree, so if the firmware should be able to change them, it should be something like:
/soc/clocks/clock@0/clock-frequency
There's also this _mmc-bcm2835_ driver coming up, so I don't know what that clock will be called.
In ARCH_BCM2835 the uart0 clock frequency is set in [clk-bcm2835](http://lxr.free-electrons.com/source/drivers/clk/clk-bcm2835.c), but I don't see why it can't be set in the Device Tree instead. The DT clocks are parsed before the driver is probed.

As for the others, I think we have come to an agreement upstream first, before we decide anything. This being part of an upstreaming effort.

Bottom line: I think we can wait with the bootloader behaviour for now.

> Do any machine/driver files need updating?

No

**Comment by notro** [comment]
@popcornmix 
Can you pull this please?
